sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/ReadmeSamples.java

```diff
@@ -196,6 +196,86 @@ public void recognizeReceipt() {
         }
     }
 
+    public void recognizeBusinessCard() {
```

Comment:
nit: this looks like it's using recognizeBusinessCardFromUrl, can you make that the name of this sample?